packages/client/src/services/credit/creditRfqRequests.ts

```diff
@@ -1,27 +1,37 @@
 import { createSignal } from "@react-rxjs/utils"
 import { filter, map, tap, withLatestFrom } from "rxjs/operators"
 
-import { showRfqInSellSide } from "@/client/utils"
+import { multiplyBy1000, showRfqInSellSide } from "@/client/utils"
```

Comment:
hehe .. bit literal, that name 😉
I was thinking of maybe something like `adjustUserCreditQuantity`, which is not a great name 😄 but along the lines of that tells the code reader what we are up to